go.mod

```
@@ -3,17 +3,17 @@ module github.com/topolvm/pvc-autoresizer
 go 1.17
```

```suggestion
go 1.18
```
If you don't have to use go v1.17 especially, I think it is better to update it.
https://go.dev/ref/mod#go-mod-file-go

---

I think it is not required because TopoLVM does not use any Go 1.18 features.
cf. https://go.dev/ref/mod#go-mod-file-go